### ISSUE TITLE
Watch mode is not supported in this environment

### ISSUE THREAD

**Issue by piotrkochan** [expo/expo-cli#1589]
## 🐛 Bug Report

INFO
22:54
Watch mode is not supported in this environment

Watchman installed, not using production mode.

### Environment

Expo CLI 3.13.1 environment info:
    System:
      OS: macOS 10.15.3
      Shell: 5.7.1 - /bin/zsh
    Binaries:
      Node: 12.14.0 - ~/.nvm/versions/node/v12.14.0/bin/node
      npm: 6.13.4 - ~/.nvm/versions/node/v12.14.0/bin/npm
      Watchman: 4.9.0 - /usr/local/bin/watchman
    IDEs:
      Xcode: /undefined - /usr/bin/xcodebuild
    npmPackages:
      expo: ~36.0.0 => 36.0.2 
      react: ~16.9.0 => 16.9.0 
      react-native: https://github.com/expo/react-native/archive/sdk-36.0.0.tar.gz => 0.61.4 
    npmGlobalPackages:
      expo-cli: 3.13.1

### Expected Behavior

App auto reload

### Actual Behavior

Watch mode is not supported in this environment both in cli result and the developer tools

**Comment by dprslt** [comment]
We go the same issue here ..

**Comment by stormsamurai1** [comment]
Same issue here, I scan the QRCode and "package is not running at 192.168.0.14:19001" appear

**Comment by martyni8** [comment]
same issue here as well :P

**Comment by maaznadeem246** [comment]
same is happening with me.
it is showing me "Watch mode is not supported in this environment"

**Comment by mgj** [comment]
Getting this on windows 10 as well

node: v12.16.1
npm: 6.13.4¨
expo: 3.13.1

**Comment by martyni8** [comment]
I was using previous versions of expo (sdk 34, not the latest) and node modules and my app is back to normal, working now. It's gonna be a version problem...

**Comment by marcoacevey** [comment]
cant solve this problem yet.

**Comment by Beam2232** [comment]
Has anyone managed to solve the problem?

**Comment by olatollu** [comment]
same problem, occurred immediately after updating expo, has it been resolved ?

**Comment by marcoacevey** [comment]
> same problem, occurred immediately after updating expo, has it been resolved ?

I tryed  to down grade the version, but it did not work for me.

**Comment by sgardn** [comment]
Just got this with a fresh creation of a new app using `expo init`, unexciting package info for reference:

```
  },
  "dependencies": {
    "@expo/vector-icons": "~10.0.0",
    "@react-native-community/masked-view": "0.1.5",
    "@react-navigation/bottom-tabs": "^5.0.0",
    "@react-navigation/native": "^5.0.0",
    "@react-navigation/stack": "^5.0.0",
    "@react-navigation/web": "~1.0.0-alpha.9",
    "expo": "~36.0.0",
    "expo-asset": "~8.0.0",
    "expo-constants": "~8.0.0",
    "expo-font": "~8.0.0",
    "expo-web-browser": "~8.0.0",
    "react": "~16.9.0",
    "react-dom": "~16.9.0",
    "react-native": "https://github.com/expo/react-native/archive/sdk-36.0.0.tar.gz",
    "react-native-gesture-handler": "~1.5.0",
    "react-native-safe-area-context": "0.6.0",
    "react-native-screens": "2.0.0-alpha.12",
    "react-native-web": "~0.11.7"
  },
  "devDependencies": {
    "@babel/core": "^7.0.0",
    "babel-preset-expo": "~8.0.0",
    "jest-expo": "~36.0.1"
  },
}```

**Comment by marcoacevey** [comment]
I downgraded to the sdk to version 34.0.2 and now its worcking fine.

**Comment by marcoacevey** [comment]
I followed these steps:
- in app.json changed:
  1) "sdkVersion": "36.0.0"  to "sdkVersion": "34.0.0";
- in package.json changed:
 1) "expo": "~36.0.0" to "expo": "~34.0.0";
 2) "react-native": "https://github.com/expo/react-native/archive/sdk-34.0.2.tar.gz";
And then run "yarn install" or "npm intall" and  now it every thing working fine :)

**Comment by Itamar05** [comment]
> I followed these steps:
> 
> * in app.json changed:
>   
>   1. "sdkVersion": "36.0.0"  to "sdkVersion": "34.0.0";
> * in package.json changed:
> 
> 1. "expo": "~36.0.0" to "expo": "~34.0.0";
> 2. "react-native": "https://github.com/expo/react-native/archive/sdk-34.0.2.tar.gz";
>    And then run "yarn install" or "npm intall" and  now it every thing working fine :)

**Comment by Itamar05** [comment]
its dont wor

> I followed these steps:
> 
> * in app.json changed:
>   
>   1. "sdkVersion": "36.0.0"  to "sdkVersion": "34.0.0";
> * in package.json changed:
> 
> 1. "expo": "~36.0.0" to "expo": "~34.0.0";
> 2. "react-native": "https://github.com/expo/react-native/archive/sdk-34.0.2.tar.gz";
>    And then run "yarn install" or "npm intall" and  now it every thing working fine :)



> I followed these steps:
> 
> * in app.json changed:
>   
>   1. "sdkVersion": "36.0.0"  to "sdkVersion": "34.0.0";
> * in package.json changed:
> 
> 1. "expo": "~36.0.0" to "expo": "~34.0.0";
> 2. "react-native": "https://github.com/expo/react-native/archive/sdk-34.0.2.tar.gz";
>    And then run "yarn install" or "npm intall" and  now it every thing working fine :)

**Comment by brentvatne** [comment]
you can add this to your package.json if you use yarn:

```
  "resolutions": {
    "@react-native-community/cli": "3.0.4"
  }
```

then run `yarn` again.

<hr />

I have reached out to the react-native-community/cli team and asked them to revert the changes that they rolled out in a minor version bump

**Comment by thymikee** [comment]
CLI was patched in v3.2.1 to make this message gone. You'll need to regenerate the lock file (e.g. yarn.lock). [Instructions here](https://github.com/react-native-community/cli#updating-the-cli).

**Comment by abdi4** [comment]
I’m getting this too and some unexpected bugs like Alert.alert immediately closing expo. Are these issues related by any chance

**Comment by brentvatne** [comment]
@abdi4 - not related, please create a new issue with the issue template completed thanks!

**Comment by gregjw** [comment]
Anyone found a solution to this yet?

**Comment by JeremieTouze** [comment]
Updating expo, in the project folder, with following command worked for me: 
`expo update`

**Comment by gregjw** [comment]
That doesn't work for me.

**Comment by JeremieTouze** [comment]
Hello @gregjw, to be more accurate, I did the [instructions ](https://github.com/react-native-community/cli#updating-the-cli )on the link provided by @thymikee, then performed the `expo update` . I'm not sure though how these steps are related, I hope this could help anyway.

**Comment by hspacheco** [comment]
I had the same message using WSL. After removing package-lock and running the expo update command the problem didn't happen again

**Comment by brentvatne** [comment]
closing and locking this because of the fix & instructions @thymikee provided above:

> CLI was patched in v3.2.1 to make this message gone. You'll need to regenerate the lock file (e.g. yarn.lock). [Instructions here](https://github.com/react-native-community/cli#updating-the-cli).
